Gemfile.lock

```diff
@@ -13,7 +13,7 @@ PATH
       parser (>= 3.1)
       pathname (>= 0.2.1)
       rainbow (>= 2.2.2, < 4.0)
-      rbs (>= 2.8.0)
+      rbs (> 2.8.0, <= 2.8.1)
```

Comment:
rbs-2.8.1 is the most compatible version of 2.8.x with Steep. `>= 2.8.2` has a regression that fails CI.